shallowblue_rllib.py

```diff
@@ -156,7 +169,13 @@ def step(self, action):
         # Get Reward
         reward = 0
         for r in world_state.rewards:
-            reward += r.getValue()
+            val = r.getValue()
+            if val == self.tnt_reward:
```

Comment:
if multiple tnt or red stone blocks are touched in the same step, this will NOT count them. Not sure how often that happens but i dont think its worth trying to create a more complicated solution